frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,46 @@
+import { Grade } from "../APIClients/types/UserClientTypes";
+
 export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
 
+export enum Type {
+  Beginning = "BEGINNING",
+  End = "END",
+}
+
 export type AssessmentTypes = {
   status: StatusProperty;
   name: string;
-  grade: string;
-  type: "BEGINNING" | "END";
+  grade: Grade;
+  type: Type;
   country: string;
   region: string;
 };
+
+export const gradeOptions = [
+  { value: "K", label: "Kindergarten" },
+  { value: "Grade 1", label: "Grade 1" },
+  { value: "Grade 2", label: "Grade 2" },
+  { value: "Grade 3", label: "Grade 3" },
+  { value: "Grade 4", label: "Grade 4" },
+  { value: "Grade 5", label: "Grade 5" },
+  { value: "Grade 6", label: "Grade 6" },
+  { value: "Grade 7", label: "Grade 7" },
+  { value: "Grade 8", label: "Grade 8" },
+];
+
+export const typeOptions = [
+  { value: "BEGINNING", label: "Beginning" },
```

Comment:
these `value`s should also use the enum i believe

Reply:
Done!

Reply:
was this updated?
```suggestion
  { value: TestTypes.Beginning, label: "Beginning" },
```